docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -117,19 +135,16 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Bad Station Deployment
 
-It seems like your station needs some improvement in terms of how and where it is deployed.
-Check out our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) for some inspiration on how and where you should install your weather station.
+It appears that your weather station could benefit from some improvements in terms of its deployment. The [Quality of Data (QoD) algorithm](/project/quality-of-data) evaluates all data packet received by a weather station every day, so a proper installation becomes mandatory to receive the maximum amount of rewards.
+We recommend that you take a look at our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) for inspiration on how and where to install your station.
+
 
-It is mandatory to have a proper installation to receive the maximum amount of rewards.
-Our QoD algorithm runs every day to evaluate your data. You can read more about this algorithm [here](/project/quality-of-data).
 
 <hr/>
 
 ## No location Data
 
-Your station did not report any GPS location and was suspended from rewards because it didn't send any GPS data.
-
-Please make sure to check the troubleshooting guides depending on your weather station before contacting our [support team](https://support.weatherxm.com). 
+Your weather station was suspended from rewards because it did not report GPS data. Please check the troubleshooting guides for your station below, before contacting our support team.[support team](https://support.weatherxm.com). 
 
 ### For the WS-1000 (WiFi/M5 version)
```

Comment:
`For the M5 (WG1000 - WiFi gateway)`

M5 is the friendly name of this product, WG1000 is the SKU (WS1000 is the weather station).
We might as well start using the more up-to-date terminology which is also in line with the new products that are coming. Eg when the new gateway comes (next product - soon - you didn't hear it from me) there will no more be a single "wifi version", but there will be wifi gateways